### PR TITLE
docs: add StrongBrew to the trainer list

### PR DESCRIPTION
**Pull request by KwintenP**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

There is no link to the StrongBrew website
Issue Number: N/A

## What is the new behavior?
There is a link to the StrongBrew website

## Does this PR introduce a breaking change?
```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -181,7 +181,7 @@
             "rev": true,
             "title": "Amexio Canvas Web Based Drag and Drop IDE by MetaMagic",
             "url": "https://amexio.tech/"
-          }	
+          }
```
Comment (StephenFluin) [inline aio/content/marketing/resources.json]: Looks like we have an unrelated change here?

**Review by StephenFluin — CHANGES_REQUESTED** [review]
There's one unrelated change that needs to be removed.

Since you'll need to update the branch anyway, can you change docs: to `docs(aio):` in your commit message? If you still have the branch locally you can just make the changes and then `git commit -am --amend` to amend the commit and then `git push -f` to rewrite your branch. You shouldn't need another PR.

File: aio/content/marketing/resources.json [hunk 2]
```diff
@@ -181,7 +181,7 @@
...
```
Comment (brandonroberts) [inline aio/content/marketing/resources.json]: Remove the whitespace change

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: aio/content/marketing/resources.json (hunk 2 above)
Comment (brandonroberts) [reply aio/content/marketing/resources.json]: Sure

Comment (trotyl) [reply aio/content/marketing/resources.json]: CC. @StephenFluin per https://github.com/angular/angular/pull/24891#discussion_r203389477

**Review by StephenFluin — APPROVED** [review]

File: aio/content/marketing/resources.json (hunk 2 above)
Comment (KwintenP) [reply aio/content/marketing/resources.json]: Removed it.

**Comment by KwintenP** [comment]
My apologies for the late response, I was on holiday.

Updated as requested. Removed the tab (which my editor did itself, hence the change). 

Thanks!

File: aio/content/marketing/resources.json (hunk 1 above)
Comment (KwintenP) [reply aio/content/marketing/resources.json]: As discussed further, the trailing tab can be removed.

**Review by brandonroberts — APPROVED** [review]
